### PR TITLE
Need akka-slf4j on classpath for internal logging

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/logging.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ For historical reasons logging by the Akka internals and by classic actors are p
 through an event bus. Such log events are processed by an event handler actor, which then emits them to
 SLF4J or directly to standard out.
 
-When `akka-actor-typed` is on the classpath this event handler actor will emit the events to SLF4J.
+When `akka-actor-typed` and `akka-slf4j` are on the classpath this event handler actor will emit the events to SLF4J.
```
[the remark this answers is not in the document]
Comment (patriknw) [reply akka-docs/src/main/paradox/typed/logging.md]: it's a dependency of akka-actor-testkit-typed, and that's probably why I haven't seen it

[the remark this answers is not in the document]
Comment (patriknw) [reply akka-docs/src/main/paradox/typed/logging.md]: added dependency in https://github.com/akka/akka/pull/28130

I don't mind merging this PR also, it's technically correct

[the remark this answers is not in the document]
Comment (chbatey) [reply akka-docs/src/main/paradox/typed/logging.md]: :+1: doing samples was when i first noticed it, i assumed it just "worked" up until that point

**Review by patriknw — APPROVED** [review]
LGTM

**Review by raboof — COMMENTED** [review]
I'm not opposed, but I'm not sure we still need this now that #28130 is in?

**Comment by patriknw** [comment]
> I'm not opposed, but I'm not sure we still need this now

I don't expect many to pay attention to this note, but if there are some dependency trouble it can be good to have the extra clarification how it works (it's reference docs after all).

**Pull request merged**